ui/tests/acceptance/open-api-path-help-test.js

```diff
@@ -0,0 +1,593 @@
+import { module, test } from 'qunit';
+import Model from '@ember-data/model';
+import { setupApplicationTest } from 'vault/tests/helpers';
+import authPage from 'vault/tests/pages/auth';
+import { deleteAuthCmd, deleteEngineCmd, mountAuthCmd, mountEngineCmd, runCmd } from '../helpers/commands';
+import openApiDrivenAttributes from '../helpers/openapi-driven-attributes';
+
+/**
+ * This set of tests is for ensuring that backend changes to the OpenAPI spec
+ * are known by UI developers and adequately addressed in the UI. When changes
+ * are detected from this set of tests, they should be updated to pass and
+ * smoke tested to ensure changes to not break the GUI workflow.
+ */
+module('Acceptance | OpenAPI path help test', function (hooks) {
+  setupApplicationTest(hooks);
+  hooks.beforeEach(function () {
+    this.pathHelp = this.owner.lookup('service:pathHelp');
+    this.newModel = Model.extend({});
+    return authPage.login();
+  });
+
+  module('engine: ssh role', function (hooks) {
+    hooks.beforeEach(async function () {
+      this.backend = 'ssh-openapi';
+      await runCmd(mountEngineCmd('ssh', this.backend), false);
+    });
+    hooks.afterEach(async function () {
+      await runCmd(deleteEngineCmd(this.backend), false);
+    });
+
+    test('getProps returns correct model attributes', async function (assert) {
+      const helpUrl = `/v1/${this.backend}/roles/example?help=1`;
+      const result = await this.pathHelp.getProps(helpUrl, this.backend);
+      assert.deepEqual(result, openApiDrivenAttributes.sshRole);
```

Comment:
Something curious is happening with github actions and it's only catching the first failure from this commit (both tests failed locally, though...). Here's the [diff](https://github.com/hashicorp/vault/pull/23622/commits/a8162576ae33597eeb203cd6278cef602e9b42b4) and only one test failed [here](https://github.com/hashicorp/vault/actions/runs/6488825772?pr=23622)